src/pages/test-result/test-result.ts

```diff
@@ -19,15 +20,45 @@ export class TestResultPage {
     [key: string]: IFaultSummary;
   };
 
+  // audio related
```

Comment:
All this audio stuff could go in your provider if you wanted

Reply:
Provider has been added now :)